src/formatDate.js

```diff
@@ -50,7 +50,47 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let indexYearsTo;
+  let yearsToLength;
+
+  const separatorFrom = fromFormat[fromFormat.length - 1];
```

Comment:
```suggestion
  const separatorFrom = fromFormat[3];
```
тут можно явно указать индекс

Reply:
done